frontend/packages/dev-console/integration-tests/support/pages/common.ts

```diff
@@ -0,0 +1,9 @@
+import { helmPO } from '../pageObjects/helm-po';
+
+export const sidebarPage = {
+  verifyChartVersion: () =>
+    cy
+      .get(helmPO.sidebarPage.chartVersion)
+      .eq(0)
+      .should('have.text', '0.2.1'),
+};
```

Comment:
Please move this function to pages/helm folder

Reply:
I have moved to frontend/packages/dev-console/integration-tests/support/pages/add-flow/catalog-page.ts, as this pane is common within catalog page. please let me know if this is okay for you. @makambalaji

Reply:
Also removed frontend/packages/dev-console/integration-tests/support/pages/common.ts

Reply:
done, https://github.com/shvasude/console/commit/0e875a464a8fd0a089b59ee6809a182c9c859cab